packages/microcosm-dom/src/action-button.js

```diff
@@ -15,7 +15,7 @@ export function generateActionButton(createElement, Component) {
     }
 
     componentWillUnmount() {
-      this.queue.cancel()
+      this.queue.complete()
```

Comment:
I don't want to cancel. That would cancel any outstanding actions, which might not be what the user wants. If that is the desired behavior, you can now do that yourself.